src/tenant/tenant.service.ts

```
@@ -1,307 +1,325 @@
-                `Error: ${errorMessage}`,
-                apiId
-            )
-            return APIResponse.error(
-                response,
-                apiId,
-                API_RESPONSES.INTERNAL_SERVER_ERROR,
-                errorMessage,
-                HttpStatus.INTERNAL_SERVER_ERROR
-            );
-        }
+      if (result && result.affected && result.affected > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)